recipy/log.py

```diff
@@ -95,6 +96,30 @@ def log_init():
     sys.excepthook = log_exception
 
 
+def log_values(custom_values=None, **kwargs):
+    """ Log a dictionary of custom value-key pairs into the database
```

Comment:
Given that we can now use keyword arguments, could you update this to remove 'a dictionary of' so it fits the more generic usage.